src/UI/class-recommended-widget.php

```diff
@@ -18,6 +18,18 @@
 
 /**
  * Provides a widget with Parse.ly recommended articles.
+ *
+ * @phpstan-type WidgetSettings array{
+ *   title: string,
+ *   return_limit: int,
+ *   display_direction: string,
+ *   published_within: int,
+ *   sort: string,
+ *   boost: string,
+ *   personalize_results: bool,
+ *   img_src: string,
+ *   display_author: bool,
+ * }
```

Comment:
Define structure for array so that PHPStan can enforce typings.

Source: https://phpstan.org/writing-php-code/phpdoc-types#array-shapes